galite-core/src/main/kotlin/org/kopi/galite/list/VIntegerCodeColumn.kt

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2013-2020 kopiLeft Services SARL, Tunis TN
+ * Copyright (c) 1990-2020 kopiRight Managed Solutions GmbH, Wien AT
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 2.1 as published by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package org.kopi.galite.list
+
+import org.kopi.galite.util.base.InconsistencyException
+
+class VIntegerCodeColumn : VCodeColumn {
+
+  /**
+   * Constructs a list column.
+   */
+  constructor(title: String, column: String, names: Array<String>, codes: Array<Int>,
+              sortAscending: Boolean) : super(title, column, names, sortAscending) {
+    this.codes = codes
+  }
+
+  /**
+   * Constructs a list column.
+   *
+   */
+  constructor(title: String, column: String, names: Array<String>, codes: IntArray,
+              sortAscending: Boolean) : this(title, column, names, makeObjectArray(codes), sortAscending)
```

Comment:
The purpose of this constructor is to wrap the java primitif type int to Integer with Integer[] makeObjectArray(int[] input) method , but we dont have primitif types in Kotlin , we only have Int , so do we really need it ?